.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,59 @@
+on: workflow_dispatch
+
+name: Integration tests
+
+jobs:
+  test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Configure AWS Credentials
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: us-west-1
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.12.25
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install jq
+
+      - name: Start cluster
+        run: |
+          pushd terraform/aws
+          export EXECUTION_TAG=$(git describe --exact-match --tags 2>/dev/null || git rev-parse --short HEAD)
+          terraform workspace new integration-tests-$EXECUTION_TAG
+          ./dag autostart 3
```

Comment:
I am wondering if it should be always 3 🤔 

But let's eventually change that if we need

Reply:
Changed to parameterized value, with default of 3.